onboarding/source/amb_light_service.c

```diff
@@ -33,21 +47,62 @@ static void lightServiceTask(void * pvParameters);
 
 obc_error_code_t initLightService(void) {
     /* USER CODE BEGIN */
-    // Create the task and queue here. Return error code if task/queue was not created successfully.
+    BaseType_t xReturned = pdFAIL;
+        if (lightServiceHandle == NULL) {
+            xReturned = xTaskCreate(lightServiceTask,
+                                    LIGHT_SERVICE_NAME,
+                                    LIGHT_SERVICE_STACK_SIZE,
+                                    NULL,
+                                    LIGHT_SERVICE_PRIORITY,
+                                    &lightServiceHandle);
+                                     }
+        if (xReturned == pdFAIL) {
+             return OBC_ERR_CODE_TASK_CREATION_FAILED;
+        
+        }
+
+        if (lightserviceQueue == NULL) {
+                lightserviceQueue = xQueueCreate(LIGHT_SERVICE_SIZE, LIGHT_QUEUE_SIZE);
```

Comment:
We use the statically-allocated version of tasks, queues, etc. In this case, use `xQueueCreateStatic` instead.